src/components/PriceLineArrow.tsx

```diff
@@ -1,17 +1,17 @@
 import React from 'react';
 import { ArrowGreenIcon, ArrowOrangeIcon } from './Icons';
-import { ARROW_HEIGHT, ARROW_COLORS, DIRECTIONS } from '../utils';
+import { DIRECTIONS } from '../utils';
 
 type TPriceLineArrowProps = {
     offScreenDirection: keyof typeof DIRECTIONS;
     color?: string;
 };
 
-const PriceLineArrow = ({ offScreenDirection, color }: TPriceLineArrowProps) => {
-    const top = offScreenDirection === DIRECTIONS.UP ? `${-ARROW_HEIGHT}px` : '';
+const PriceLineArrow = ({ offScreenDirection }: TPriceLineArrowProps) => {
+    const top = offScreenDirection === DIRECTIONS.UP ? `24px` : '-41px';
```

Comment:
```suggestion
    const top = offScreenDirection === DIRECTIONS.UP ? '24px' : `${-ARROW_HEIGHT}px`;
```
@henry-deriv please change ARROW_HEIGHT to 41 in src/utils/index.ts then. Btw where are these 41 and 24 px values coming from?😊

Reply:
<img width="1018" alt="Screenshot 2023-03-10 at 8 22 12 AM" src="https://user-images.githubusercontent.com/118344354/224192100-7e8b7fd9-0e74-453e-86ae-a2abbfa31e95.png">
<img width="994" alt="Screenshot 2023-03-10 at 8 23 13 AM" src="https://user-images.githubusercontent.com/118344354/224192111-11ba8816-bb23-4e7e-ada9-c4a14673d4c7.png">

Initially had **barrier indicator arrow** above **drag price(rectangular grey box)** if barrier was positive and **barrier indicator arrow** below **drag price** if barrier was negative. Design change required that we move barrier indicator arrow to the opposite side of drag price.

Reason why they are two different values is because of where drag price sits relative to top and bottom of our chart.
Theres a lot more white space below our charts than above.

See attached.
<img width="512" alt="Screenshot 2023-03-10 at 8 31 43 AM" src="https://user-images.githubusercontent.com/118344354/224192966-bfe3acee-5b6f-4d4b-8095-f7a05a5bd0ea.png">

Reply:
thank you for the detailed elaboration! )